include/termcolor/termcolor.hpp

```diff
@@ -193,69 +207,66 @@ namespace termcolor
         }
         return stream;
     }
-
-    template <uint8_t code> inline
-    std::ostream& color(std::ostream& stream)
+    
+    template <uint8_t code, typename CharT>
+    inline
+    std::basic_ostream<CharT>& color(std::basic_ostream<CharT>& stream)
     {
         if (_internal::is_colorized(stream))
         {
         #if defined(TERMCOLOR_USE_ANSI_ESCAPE_SEQUENCES)
-            char command[12];
-            std::snprintf(command, sizeof(command), "\033[38;5;%dm", code);
-            stream << command;
+            stream << "\033[38;5;" << +code << "m";
```

Comment:
Just curious, why do we need these plus signs here and below?

Reply:
to read char as integer, instead of character